src/pybind/mgr/dashboard/README.rst

```diff
@@ -69,6 +69,42 @@ The password will be stored as a hash using ``bcrypt``.
 
 The Dashboard's WebUI should then be reachable on TCP port 8080.
 
+Enabling the Object Gateway management frontend
+-----------------------------------------------
+
+If you want to use the Object Gateway management functionality of the
+dashboard, you will need to provide credentials. If you do not have a user
+which shall be used for providing those credentials, you will also need to
+create one::
+
+  $ radosgw-admin user create --uuid=<user> --display-name=<display-name> \
+      --system
+
+The credentials of a user can also be obtained by using `radosgw-admin`::
+
+  $ radosgw-admin user info --uid=<user>
+
+Finally, set the credentials to the dashboard module::
+
+  $ ceph dashboard set-rgw-api-secret-key <secret_key>
```

Comment:
Consider changing the order of these commands, as the access key is printed before the secret key in the JSON returned by `radosgw-admin user info`